pyproject.toml

```
@@ -170,14 +171,22 @@ max-attributes = 12
 # has the issue was completed.
 # Please see this issue for more details:
 # https://github.com/frequenz-floss/frequenz-sdk-python/issues/982
-addopts = "-W=all -Werror -Wdefault::DeprecationWarning -Wdefault::PendingDeprecationWarning -Wdefault::pytest.PytestUnraisableExceptionWarning -vv"
+addopts = "-Wdefault::pytest.PytestUnraisableExceptionWarning -vv"
```

```suggestion
addopts = "-vv"
```

All warnings config should be moved to the `filterwarnings` option.